src/server/index.ts

```diff
@@ -242,6 +242,10 @@ initDb()
 
     const port = 8080
     app.listen(port, () => logger.info(`Listening on port ${port}!`))
+
+    process.on('unhandledRejection', (error) => {
+      logger.error(`Unhandled rejection:\t${error}`)
```

Comment:
what do you think about sending this event to DD too as an alert of sorts? seems like we want to be more explicitly notified of unhandled rejections

Reply:
sounds a good idea to have DD alerts for this! it looks to me that there's 2 options for that (that don't take too much extra effort):
1. no changes to the above code, create a DD monitor based on logs filtering with the term "Unhandled rejection"
2. add a new metric here for unhandled rejections, create a DD monitor based on the metric

(2) feels a bit more proper, so have a slight preference for that but am ok with whichever approach, what do you think?

Reply:
I would prefer (2) too

Reply:
have updated, please take a look